source/extensions/quic_listeners/quiche/active_quic_listener.cc

```diff
@@ -220,6 +221,12 @@ ActiveQuicListenerFactory::ActiveQuicListenerFactory(
           : 20000;
   quic_config_.set_max_time_before_crypto_handshake(
       quic::QuicTime::Delta::FromMilliseconds(max_time_before_crypto_handshake_ms));
+  // TODO(danzh) defer setting flow control window till getting filter chain. This requires
+  // QUICHE support to set the session's flow controller after instantiation.
+  quic_config_.SetInitialStreamFlowControlWindowToSend(
+      2 * Http2::Utility::OptionsLimits::MIN_INITIAL_STREAM_WINDOW_SIZE);
```

Comment:
If this isn't configurable, shouldn't we default to DEFAULT_INITIAL_STREAM_WINDOW_SIZE?